commands/shell.go

```diff
@@ -0,0 +1,46 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+// ExecuteShell returns a cobra command to execute a line through the system shell.
+// This uses the os/exec package to execute the command. The default command name is `!`.
+func ExecuteShell() *cobra.Command {
+	shellCmd := &cobra.Command{
+		Use:                "!",
+		Short:              "Execute the remaining arguments with system shell",
+		DisableFlagParsing: true,
+		RunE: func(_ *cobra.Command, args []string) error {
+			if len(args) == 0 {
```

Comment:
```suggestion
			if len(args) < 2 {
```

See

https://github.com/reeflective/console/pull/58#discussion_r1844996541

https://github.com/reeflective/console/pull/58#discussion_r1844996301